third_party/gfootball_engine/CMakeLists.txt

```diff
@@ -77,13 +77,14 @@ endif()
 
 message("Using python_boost: ${BOOST_PYTHON_VERSION}")
 
-FIND_PACKAGE(Boost REQUIRED COMPONENTS system thread filesystem ${BOOST_PYTHON_VERSION})
+FIND_PACKAGE(Boost REQUIRED COMPONENTS system filesystem ${BOOST_PYTHON_VERSION})
+FIND_PACKAGE(Boost COMPONENTS thread)
```

Comment:
I think this is no longer necessary and can be moved back to where it was declared originally.